package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A source plugin for Gatsby that consumes the JSON API of the bitcrowd CMS.",
   "main": "index.js",
   "scripts": {
-    "build": "babel src --out-dir . --ignore **/__test__ --ignore **/*.test.js",
+    "build": "babel src --out-dir . --ignore \"**/__test__\",\"**/*.test.js\"",
```

Comment:
This one gave me a good headache... 🤦‍♂

babel only accepts the first `--ignore` option and discards others, which caused the `__test__` directory to be ignored, but the `src/*test.js` files to be compiled to the root directory together with the rest of the code. Which in turn caused jest to try to run them, and `gatsby-node.test.js` to fail to import the test fixtures (as they hadn't been copied to the top-level folder).